libs/FirmwareKit/CMakeLists.txt

```
@@ -4,9 +4,13 @@
 
 add_library(FirmwareKit STATIC)
 
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/include/os_version.h.in"
+	"${CMAKE_CURRENT_BINARY_DIR}/include/os_version.h")
+
```

The `.in` file can be moved in another folder, for example

```suggestion
configure_file("${ROOT_DIR}/cmake/templates/LekaOS/os_version.h.in"
	"${CMAKE_CURRENT_BINARY_DIR}/include/os_version.h")
```

---

Conclusion: `os_version` is only used by FirmwareKit, so it can stay here